src/runJscodeshiftCodemod.ts

```diff
@@ -42,6 +35,22 @@ const transform = (
 		__INTUITA__options: options,
 	});
 
+	try {
+		const codeToExecute = `
+			${codemodSource}
+
+			transform(__INTUITA__file, __INTUITA__api, __INTUITA__options);
```

Comment:
I would prefer if the codemod registry ensures that the exported function is called `transform` (using linting?), but if we need to support both, then I would search to codemod source for `transformer` and `transform` phrases
If `transform` is present, use it, otherwise use `transformer`. If none are found, abort.
You can pass the function name using the `${functionName}` notation.